### PR TITLE
fix comma deprecation warning

### PR DESCRIPTION
**Pull request by stefan-koch-sociomantic**
Nothing much more to say.
Just fixing the warning.

### CODE REVIEW

**Comment by mihails-strasuns-sociomantic** [comment]
Would you mind adding some trivial unittest for that method while you are at it?

**Comment by stefan-koch-sociomantic** [comment]
@mihails-strasuns-sociomantic No, I'll add a unittest as soon as I know what it does.
(I just applied the fix mechanically)

File: src/ocean/net/Uri.d [hunk 1]
```diff
@@ -552,7 +552,7 @@ class Uri : UriView
         static mstring encode (cstring text, int flags)
         {
                 void[] s;
-                encode ((Const!(void)[] v) {return s ~= v, v.length;}, text, flags);
+                encode ((Const!(void)[] v) {s ~= v;return  v.length;}, text, flags);
```
Comment (mathias-lang-sociomantic) [inline src/ocean/net/Uri.d]: Nitpick, but missing a space between `;` and `return`.

**Review by leandro-lucarella-sociomantic — COMMENTED** [review]
Also nitpick, but can you capitalize the first word of the commit message? :) Adding a more extended description of what is the warning about might be helpful too (I guess is a compiler deprecation message, but just to be sure). Like saying starting from which versions this was deprecated and such might be informative.

**Comment by stefan-koch-sociomantic** [comment]
@leandro-lucarella-sociomantic Done.

**Review by nemanja-boric-sociomantic — APPROVED** [review]

**Comment by mihails-strasuns-sociomantic** [comment]
> No, I'll add a unittest as soon as I know what it does

As only `Uri.toString` method is affected, I expect adding test should be trivial. If it proves complicated for any reason, don't bother - just mention so and I'll merge.

**Comment by stefan-koch-sociomantic** [comment]
@mihails-strasuns-sociomantic added `toString()` to unittest

**Review by mihails-strasuns-sociomantic — APPROVED** [review]
Thanks!

**Comment by mihails-strasuns-sociomantic** [comment]
Looks like that unittest is failing :)

File: src/ocean/net/Uri.d [hunk 2]
```diff
@@ -968,18 +968,19 @@ unittest
...
         test(port == InvalidPort);
         test(userinfo == null);
         test(fragment == null);
         test(path == "/click.html/c=37571:RoS_Intern_search-link3_LB_Sky_Rec/b=98983:news-time-search-link_leader_neu/l=68||||de/url=http://ads.ad4max.com/adclick.aspx");
         test(query == "id=cf722624-efd5-4b10-ad53-88a5872a8873&pubad=b9c8acc4-e396-4b0b-b665-8bb3078128e6&avid=963171985&adcpc=xrH/+xVeFaPVkbVCMufB5A==&a1v=6972657882&a1lang=de&a1ou=http://ad.search.ch/iframe_ad.html?campaignname=RoS_Intern_search-link3_LB_Sky_Rec%26bannername=news-time-search-link_leader_neu%26iframeid=sl_if1%26content=vZLLbsIwEEX3+Qo3aqUW1XEgkAckSJRuKqEuoDuELD+miSEJyDEE/r7h0cKm6q6SF9bVjH3uzI3vMOaVYdpgPIwrodXGIHPYQGIb2BuyZDt2Vu2hRUh8Nx+/jj5Gc4u0UNAJ95H7jCbAJGi+Zlqix3eoqyfUIhaT3YLtabpVEiXI5pEImRBdDF7k4y53Oea+38Mh554bhO1OCL49+O6qlTTZsa3546pmoNLMHOXIvaoapNIgTnpmzKZPCJNOBUyLzBEZEbkSKyczRU5E4gW9oN2frmf0rTSgS3quw7kqVx6dvNDZ6kCnIAhPojAKvX7Z+MFGFYBvKml+skxL2JI88cOHYPxzJJCtzpP79pXQaCZWqkxppcVvlDsF9b9CqiJNLiB1Xd+QqIKlUBHXSdWnjQbN1heLoRWTcwz+CAlqLCZXg5VzHoEj1gW5XJeVffOcFR8TCKVs8vcF%26crc=ac8cc2fa9ec2e2de9d242345c2d40c25");
-
+        test(toString() == uristring);
```
Comment (mathias-lang-sociomantic) [inline src/ocean/net/Uri.d]: `test!("==")(toString(), uristring);`

File: src/ocean/net/Uri.d [hunk 3]
```diff
@@ -989,6 +990,7 @@ unittest
         test(fragment == "_presence");
         test(path == "/~marenz");
         test(query == "what");
+        test(toString() == "psyc://example.net/~marenz?what#_presence");
```
Comment (mathias-lang-sociomantic) [inline src/ocean/net/Uri.d]: Ditto `test!("==")`

**Comment by stefan-koch-sociomantic** [comment]
@mihails-strasuns-sociomantic It fails because the toString escapes `?` and `=`

**Comment by stefan-koch-sociomantic** [comment]
fixed by removing the offending unittest.

**Comment by mihails-strasuns-sociomantic** [comment]
Thanks! Btw I recommend adding chrome/firefox extension for codecov to be able to easily check own PR test coverage (chrome one works much better)

**Pull request merged**